test/cabbage/missing_step_advisor_test.exs

```diff
@@ -0,0 +1,90 @@
+defmodule Cabbage.MissingStepAdvisorTest do
+  use ExUnit.Case, async: true
+
+  alias Cabbage.MissingStepAdvisor, as: Sut
+
+  test "raises RuntimeError if step is missing" do
+    step_text = "I am Bob"
+    step_type = "Given"
+    expected_message = """
+    Please add a matching step for:
+    "Given I am Bob"
+
+      defgiven ~r/^I am Bob$/, vars, state do
+        # Your implementation here
+      end
+    """
+
+    assert_raise(RuntimeError,
```

Comment:
Perhaps we should change this to a missing step error and use Elixir's `defexception` to handle all this for us...  https://hexdocs.pm/elixir/Kernel.html#defexception/1